bots/inspect-queue

```diff
@@ -21,38 +21,57 @@ MAX_PRIORITY = 9
 
 import argparse
 import sys
-import time
+import ssl
 
-import amqp
+import pika
+
+seen_messages = 0
 
 def main():
     parser = argparse.ArgumentParser(description='Read and print messages from the queue without acknowleding them')
     parser.add_argument('--amqp', default='localhost',
                         help='The URL of the AMQP server to consume from')
     parser.add_argument('--queue', default='tasks',
                         help='The name of the queue to consume from')
-    parser.add_argument('-n', '--number', default=0,
-                        help='Number of queue items to show, starting from the front (0=all)')
-    parser.add_argument('-t', '--timeout', default=0.5,
-                        help='Time to wait for messages')
     opts = parser.parse_args()
 
-    connection = amqp.Connection(host=opts.amqp)
-    connection.connect()
+    host, port = opts.amqp.split(':')
```

Comment:
This now  looks like `--amqp` would consist of a host:port, not an URL any more. Thus the help text should be adjusted. It's also useful to include `(default: %(default)s)` in the help text.

This will also crash if the given value does not contain a `:`, in particular when using the default.